scripts/create-new-module

```diff
@@ -80,12 +86,19 @@ fs.writeFileSync(path.join(folderPath, 'package.json'), `{
   },
   "homepage": "https://github.com/Turfjs/turf",
   "devDependencies": {
+    "@std/esm": "*",
     "benchmark": "^2.1.4",
+    "rollup": "^0.51.7",
```

Comment:
Also note, I've been changing all `devDependencies` to `*` since those dependencies are pretty stable.